doc/user-guide.adoc

```diff
@@ -400,7 +401,45 @@ spec:
 
 The link:++https://github.com/redhat-developer/service-binding-operator++[Service Binding Operator] enables application developers to bind applications together with operator-managed backing services. This can be achieved by creating a `ServiceBindingRequest` custom resource.
 
-For the Runtime Component Operator to pass the binding information to the application, you can specify the name of the `ServiceBindingRequest` custom resource as follows:
+
+For the Runtime Component Operator to pass the binding information to the application, define your `ServiceBindingRequest` custom resource in one of the following two ways:
+
+. Define the `ServiceBindingRequest` custom resource YAML within the `bindings.embedded` parameter in your `RuntimeComponent` custom resource.
+. Create the `ServiceBindingRequest` custom resource manually and either refer to it explicitly in your `RuntimeComponent` custom resource or let the operator detect it automatically.
```

Comment:
It may be good to clarify that it'll be auto detected only if the naming convention is followed.